README.md

```diff
@@ -329,9 +329,10 @@ Library specific options:
 
 It is highly recommended to use vcpkg as package manager.
 
-After you successfully built vcpkg you can easily install the required libraries:
+After you successfully built vcpkg, install the required libraries by running:
+
 ```powershell
-vcpkg install zlib zstd curl[winssl] openal-soft libvorbis libogg libjpeg-turbo sqlite3 freetype luajit gmp jsoncpp opengl-registry --triplet x64-windows
+vcpkg install
```

Comment:
I think users should still be told to specify the packages manually (also you need to keep the `--triplet ...`), the explanations in the README only make sense if you have the list of packages visible.